app/models/backend_api.rb

```diff
@@ -16,6 +17,7 @@ class BackendApi < ApplicationRecord
   delegate :default_api_backend, to: :class
 
   validates :name,        length: { maximum: 511 }, presence: true
```

Comment:
So I was wondering

Why do we have 511 characters here????

Reply:
hahahahaha good question :joy: i will check :smile:

Reply:
https://github.com/3scale/porta/blob/7131eb1fb93b6f0be507cfa575e091726a66c1e4/db/schema.rb#L235
https://github.com/3scale/porta/commit/c2204dcdf8cf3e5a9484e635df3bba7544e8b529#diff-e2de550cd7105b0cc383c49f401ca88dR6